src/embeddings.rs

```diff
@@ -111,7 +110,7 @@ impl PyEmbeddings {
     ///
     /// If the word is not known, its representation is approximated
     /// using subword units.
-    fn embedding(&self, word: &str) -> PyResult<Py<PyArray1<f32>>> {
+    fn embedding(&self, word: &str) -> PyObject {
```

Comment:
In these cases you can return an `Option` of the returned type. `None` will be mapped to `None` in Python.

Reply:
Should be fixed. There seem to be infinite ways of doing things with `pyo3`.